app/views/idv/shared/_verify.html.erb

```diff
@@ -4,84 +4,94 @@ locals:
   remote_identity_proofing - true if we're doing RIDP, false if we're doing IPP
   step_url - generator for step URLs
 %>
-<% title t('titles.doc_auth.verify_info') %>
+<% title t('titles.idv.verify_info') %>
 
-<%= render PageHeadingComponent.new.with_content(t('doc_auth.headings.verify')) %>
+<%= render PageHeadingComponent.new.with_content(t('headings.verify')) %>
 <div class='margin-top-4 margin-bottom-2'>
   <div class="grid-row grid-gap grid-gap-2 padding-bottom-1">
-    <div class='grid-col-fill'>
+    <dl class="grid-col-fill margin-bottom-0">
       <div>
-        <%= "#{t('doc_auth.forms.first_name')}: #{pii[:first_name]}" %>
+        <dt class="display-inline"> <%= t('idv.form.first_name') %>: </dt>
+        <dd class="display-inline margin-left-05"> <%= pii[:first_name] %> </dd>
```

Comment:
I took this opportunity to implement [Andrew's suggestion](https://github.com/18F/identity-idp/pull/6514#discussion_r906006828) to improve the HTML semantics of this page by using description lists

Reply:
We may not need both the `margin-left-05` and the explicit " " space, since my intention with the `margin-left-05` in the other component was as a substitute for the space character.